src/main/java/com/extendedclip/deluxemenus/config/DeluxeMenusConfig.java

```diff
@@ -763,23 +776,25 @@ private Map<Integer, TreeMap<Integer, MenuItem>> loadMenuItems(FileConfiguration
 
             // item flags
             if (c.contains(currentPath + "item_flags")) {
-                if (c.isString(currentPath + "item_flags")) {
-                    String flagAsString = c.getString(currentPath + "item_flags");
-                    ItemFlag flag;
-                    try {
-                        flag = ItemFlag.valueOf(flagAsString.toUpperCase());
-                        builder.itemFlags(Collections.singletonList(flag));
-                    } catch (IllegalArgumentException | NullPointerException ignored) {
+                List<ItemFlag> itemFlags = new ArrayList<>();
+
+                for (String flagAsString : getStringListFromConfig(c, currentPath + "item_flags")) {
+                    ItemFlag flag = Enums.getIfPresent(ItemFlag.class, flagAsString.toUpperCase()).orNull();
```

Comment:
Is there a reason to convert to nullable value? why can't we just use the Optional?

Reply:
`Enums` uses Google's [`Optional`](https://guava.dev/releases/19.0/api/docs/com/google/common/base/Optional.html) and Java's is already imported in that class and I didn't want to use the full qualifier for the variable _`com.google.common.base.Optional<ItemFlag>`_